src/builder.rs

```diff
@@ -654,24 +654,12 @@ impl<'a, 'gcc, 'tcx> BuilderMethods<'a, 'tcx> for Builder<'a, 'gcc, 'tcx> {
 
     fn load(&mut self, _ty: Type<'gcc>, ptr: RValue<'gcc>, _align: Align) -> RValue<'gcc> {
         // TODO(antoyo): use ty.
-        let block = self.llbb();
-        let function = block.get_function();
-        // NOTE: instead of returning the dereference here, we have to assign it to a variable in
-        // the current basic block. Otherwise, it could be used in another basic block, causing a
-        // dereference after a drop, for instance.
-        // TODO(antoyo): handle align of the load instruction.
-        let deref = ptr.dereference(None).to_rvalue();
-        let value_type = deref.get_type();
-        unsafe { RETURN_VALUE_COUNT += 1 };
-        let loaded_value = function.new_local(None, value_type, &format!("loadedValue{}", unsafe { RETURN_VALUE_COUNT }));
-        block.add_assignment(None, loaded_value, deref);
-        loaded_value.to_rvalue()
+        // TODO: handle non-natural alignments
```

Comment:
The rust repo requires that TODO should be annotated with a name. If you don't plan to fix this later, just put my username there.